<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditItem</class>
 <widget class="QMainWindow" name="EditItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Item Details</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>511</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>62</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Item ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="in_itemID">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="ItemDetails" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>46</number>
      </property>
      <property name="verticalSpacing">
       <number>18</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="in_title">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Extra_Label">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="in_extra">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="in_type_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="in_subject">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Item type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Subject_label">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>381</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>75</number>
      </property>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Delete Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_button">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Update Information</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
